app/src/main/res/values-night-v23/styles.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+  -->
+
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Is there any reason why the night styles are only available for API 23+? Dark mode is backwards compatible well past our minSdk (21).

Reply:
there was something about the toolbar background colour... i'll look into it